examples/build.gradle

```diff
@@ -31,22 +28,22 @@ def protobufVersion = '3.5.1'
 def protocVersion = '3.5.1-1'
 
 dependencies {
-    compile "com.google.api.grpc:proto-google-common-protos:1.0.0"
-    compile "io.grpc:grpc-alts:${grpcVersion}"
-    compile "io.grpc:grpc-netty-shaded:${grpcVersion}"
-    compile "io.grpc:grpc-protobuf:${grpcVersion}"
-    compile "io.grpc:grpc-stub:${grpcVersion}"
-    compileOnly "javax.annotation:javax.annotation-api:1.2"
```

Comment:
`compileOnly` is still valid, as I understand it: https://developer.android.com/studio/build/dependencies

Reply:
Right, I replaced it by mistake, actually I wanted to keep it.

Reply:
Done.